google/datalab/contrib/pipeline/airflow_operators/bq_load_operator.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+
+import google
```

Comment:
Should go ahead an import google.datalab.bigquery as well.

Reply:
I had been avoiding that for pipeline since I found it easier to manage without the remapping in __init__ files, so this carried over from that. Anyway, done now.